data/json/itemgroups/mansion.json

```diff
@@ -1168,15 +1165,11 @@
     "items": [
       { "group": "beer_selection", "prob": 30 },
       { "group": "glass_of_wine", "prob": 30 },
-      [ "novel_pulp", 30 ],
-      [ "mag_news", 30 ],
-      [ "mag_tv", 30 ],
+      { "group": "book_mag_gen", "prob": 120 },
       { "group": "tobacco_products", "prob": 50 },
+	  [ "novel_pulp", 30 ],
```

Comment:
This file need to be linted.

Reply:
Thanks. I delinted the files originally but had to make changes after making the PR and did not re-lint.